### ISSUE TITLE
Rendering a GLCanvas

### ISSUE THREAD

**Issue by smac89** [eclipse-efx/efxclipse-drift#13]
Is it possible to render a GlCanvas unto this surface? If so, can you give me a hint as to where to start? 

Thank you!

**Comment by tomsontom** [comment]
i guess you talk about JOGL?

**Comment by smac89** [comment]
@tomsontom Yes JOGL

**Comment by tomsontom** [comment]
Yes that is definately possible and similar to #9 but we currently can not promise when we will get to this unless someone contributes support for it.

**Comment by redrezo** [comment]
The 1.0.0.rc0 release has already a sample on how to use lwjgl (see https://github.com/BestSolution-at/efxclipse-drift-samples repo). You can use this a starting point to use jogl. If you come up with a working sample it would be nice to include it in our samples repo.
